python/ray/serve/_private/deployment_state.py

```diff
@@ -1243,48 +1271,47 @@ def _stop_wrong_version_replicas(self, max_to_stop=math.inf) -> int:
             max_replicas=max_to_stop,
             ranking_function=rank_replicas_for_stopping,
         )
-        replicas_stopped = False
+        replicas_changed = False
         code_version_changes = 0
-        user_config_changes = 0
+        reconfigure_changes = 0
         for replica in replicas_to_update:
-            # If the code version is a mismatch, we stop the replica. A new one
-            # with the correct version will be started later as part of the
+            replicas_changed = True
```

Comment:
will this trigger `_stop_wrong_version_replicas` returning True and potentially trigger a long poll broadcast which is not necessary?

Reply:
That's a good point, it's only necessary for replicas that have max concurrent queries updated. I've changed the code to only return True in that case.